Changelogs for Digital Twin Manager UI and Microservice 1020.2.0 (#2429)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020.2.0-added-subassets-module-as-a-plugin-into-dtm-plugins.md

---
date: ""

> title: "Added 'Subassets' module as a plugin into DTM plugins"

Reviewer: ```suggestion
title: "Added Subassets module as a plugin into DTM plugins"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-QHu1GdukP"
      label: "Feature"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1484"
version: "1020.2.0"
---

> The **Subassets** module is now a cumulocity official plugin. It can be integrated with default and custom cumulocity web applications to view **Subassets**.

Reviewer 2: ```suggestion
The **Subassets** module is now a {{< product-c8y >}} official plugin. It can be integrated with default and custom {{< product-c8y >}} web applications to view **Subassets**.
```

> The **Subassets** module is now a {{< product-c8y >}} official plugin. It can be integrated with default and custom {{< product-c8y >}} web applications to view subassets.

Reviewer 2: ```suggestion
The subassets module is now a {{< product-c8y >}} official plugin. It can be integrated with default and custom {{< product-c8y >}} web applications to view subassets.
```

> The **Subassets** module is now a {{< product-c8y >}} official plugin. It can be integrated with default and custom {{< product-c8y >}} web applications to view **Subassets**.

Reviewer: ```suggestion
The **Subassets** module is now a {{< product-c8y >}} official plugin. It can be integrated with default and custom {{< product-c8y >}} web applications to view subassets.
```

> The subassets module is now a {{< product-c8y >}} official plugin. It can be integrated with default and custom {{< product-c8y >}} web applications to view subassets.

Reviewer 3: @pavanshivajirao as highlighted, this doesn't seem to be a changelog entry for the replacement of sub-asset models in the DTM app with the DTM plugin's sub-asset module. Would request to correct the changelog entry.
